.chloggen/deprecate_QueueConfig.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: exporterhelper
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Deprecate `QueueConfig` in favor of `QueueBatchConfig`.
```

Comment:
Should this have include some information about how this will affect the `otlp` and `otlphttp`exporters?